app/ui/legacy/src/main/java/com/fsck/k9/ui/settings/AboutFragment.kt

```diff
@@ -142,6 +148,24 @@ class AboutFragment : Fragment() {
                 "https://github.com/splitwise/TokenAutoComplete/",
                 "Apache License, Version 2.0",
             ),
+            Library("accompanist","https://github.com/google/accompanist","Apache License, Version 2.0"),
+            Library("FlexboxLayout","https://github.com/google/flexbox-layout","Apache License, Version 2.0"),
+            Library("Error Prone","https://github.com/google/error-prone","Apache License, Version 2.0"),
+            Library("ListenableFuture",
+                "https://mvnrepository.com/artifact/com.google.guava/listenablefuture/1.0",
```

Comment:
We never want to use `mvnrepository.com` URLs. Please change this to https://github.com/google/guava instead.